pygoat/production_settings.py

```diff
@@ -1,10 +1,11 @@
+import os
+import dj_database_url
```

Comment:
We didn't end up using this package so there's no need to import it

Reply:
good call, removed.